src/startupscriptgenerator/node/scriptgenerator.go

```diff
@@ -48,7 +48,7 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 
 	logger.LogInformation("Generating script for source at '%s'", gen.SourcePath)
 
-	const oryxManifestFile string = "oryx-manifest.toml"
+	const oryxManifestFile string = consts.BuildManifestFileName
```

Comment:
why do we even need this other const, `oryxManifestFile`, if we can just use `consts.BuildManifestFileName`?

Reply:
yeah, i know...i wanted to minimize the changes...will change it now..